test/test_axes.jl

```diff
@@ -88,3 +88,33 @@ end
     p = plot(1:2, xtickor = :out)
     @test p[1][:xaxis][:tick_direction] === :out
 end
+
+@testset "aliases" begin
+    compare(p::Plots.Plot, s::Symbol, val, op::Symbol) =
+        eval(Expr(:comparison, Meta.quot(p[1][:xaxis][s]), op, Meta.quot(p[1][:yaxis][s]), op, Meta.quot(p[1][:zaxis][s]), op, Meta.quot(val)))
```

Comment:
```suggestion
    compare(p::Plots.Plot, s::Symbol, val, op) =
        op(p[1][:xaxis][s]), va) &&  op(p[1][:yaxis][s]), val) && op(p[1][:zaxis][s], val)
```
I'd avoid `eval` where I can. Seems a bit overkill to me here.

Reply:
Even better, turn `compare` to `@compare` (that's a macro job).

Reply:
Indeed this also works, and is probably even slightly faster. 
Will update the PR.

Reply:
Okay I had quite a deep look into macros and think it is not (easily) possible and the suggestion by @BeastyBlacksmith looks much clearer.
From my side the current state of this PR would be mergeable.

Let me know if you still have any suggestions or an idea how the macro implementation would work.

Reply:
Writing macros can be tricky. The current implementation is compact and readable, that's all that matters.